client/gutenberg/extensions/tiled-gallery/index.js

```diff
@@ -13,58 +11,74 @@ import { Rect, SVG } from '@wordpress/components';
 import { __, _x } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 import edit from './edit';
 import save from './save';
-import { DEFAULT_COLUMNS, DEFAULT_LAYOUT, LAYOUT_STYLES, LAYOUTS } from './constants';
+import { DEFAULT_LAYOUT, LAYOUT_STYLES } from './constants';
 
 /**
  * Style dependencies
  */
 import './editor.scss';
 
 const blockAttributes = {
+	columns: {
+		type: 'number',
+	},
+	ids: {
```

Comment:
👋 @jeherve You'll be interested in the image ID array in attributes for https://github.com/Automattic/jetpack/pull/11000